homeassistant/components/deconz/sensor.py

```diff
@@ -67,171 +70,163 @@
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+    ),
+    DeconzSensorDescription[Daylight](
+        key="daylight_status",
+        isinstance_fn=lambda device: isinstance(device, Daylight),
+        value_fn=lambda device: DAYLIGHT_STATUS[device.daylight_status],
+        update_key="status",
+        icon="mdi:white-balance-sunny",
+        entity_registry_enabled_default=False,
+    ),
+    DeconzSensorDescription[GenericStatus](
+        key="status",
+        isinstance_fn=lambda device: isinstance(device, GenericStatus),
+        value_fn=lambda device: device.status,
+        update_key="status",
+    ),
+    DeconzSensorDescription[Humidity](
+        key="humidity",
+        isinstance_fn=lambda device: isinstance(device, Humidity),
+        value_fn=lambda device: device.scaled_humidity,
+        update_key="humidity",
+        device_class=SensorDeviceClass.HUMIDITY,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+    ),
+    DeconzSensorDescription[LightLevel](
+        key="light_level",
+        isinstance_fn=lambda device: isinstance(device, LightLevel),
+        value_fn=lambda device: device.scaled_light_level,
+        update_key="lightlevel",
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=LIGHT_LUX,
+    ),
+    DeconzSensorDescription[Power](
+        key="power",
+        isinstance_fn=lambda device: isinstance(device, Power),
+        value_fn=lambda device: device.power,
+        update_key="power",
+        device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=POWER_WATT,
+    ),
+    DeconzSensorDescription[Pressure](
+        key="pressure",
+        isinstance_fn=lambda device: isinstance(device, Pressure),
+        value_fn=lambda device: device.pressure,
+        update_key="pressure",
+        device_class=SensorDeviceClass.PRESSURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PRESSURE_HPA,
+    ),
+    DeconzSensorDescription[Temperature](
+        key="temperature",
+        isinstance_fn=lambda device: isinstance(device, Temperature),
+        value_fn=lambda device: device.scaled_temperature,
+        update_key="temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=TEMP_CELSIUS,
+    ),
+    DeconzSensorDescription[Time](
+        key="last_set",
+        isinstance_fn=lambda device: isinstance(device, Time),
+        value_fn=lambda device: dt_util.parse_datetime(device.last_set),
+        update_key="lastset",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        state_class=SensorStateClass.TOTAL_INCREASING,
```

Comment:
I'm not sure I have seen this before that a timestamp device class sensor uses total increasing as state class. I don't think this is correct.

Reply:
Thanks Martin! Resolved in https://github.com/home-assistant/core/pull/79380